server/container/util-java8/src/test/java/org/apache/james/util/CompletableFutureUtilTest.java

```diff
@@ -54,4 +55,13 @@ public void allOfShouldUnboxStream() {
                 .collect(Guavate.toImmutableList()))
             .containsOnly(value1, value2, value3);
     }
+
+    @Test
+    public void allOfShouldWorkOnVeryLargeStream() {
```

Comment:
(I prefer to demonstrate that it works, so that people don't accuse my tool ;-) )